core/controllers/feedback.py

```diff
@@ -215,8 +253,18 @@ def get(self):
             feedback_services.get_next_page_of_all_feedback_messages(
                 urlsafe_start_cursor=urlsafe_start_cursor))
 
+        messages_dict = [m.to_dict() for m in all_feedback_messages]
+
+        messages_author_ids = [
+           message['author_id'] for message in messages_dict]
+
+        messages_user_names = user_services.get_usernames(messages_author_ids)
+
+        for index, message in enumerate(messages_dict):
+            message['author_username'] = messages_user_names[index]
+
```

Comment:
Again, this code is a bit similar to the code around line 193.

Reply:
I tried to change the function as per our discussion. PTAL